src/lib.rs

```diff
@@ -72,7 +72,7 @@ pub struct Nmea {
     pub pdop: Option<f32>,
     /// Geoid separation in meters
     pub geoid_separation: Option<f32>,
-    pub fix_satellites_prns: Option<Vec<u32>>,
+    pub fix_satellites_prns: Option<Vec<u32, 12>>,
```

Comment:
I suppose it should be 58, but it can wait until we can teach this crate to merge GSA messages.

Reply:
Ok, Can you also provide some info on this merging?